downstream/modules/platform/proc-create-postresql-secret.adoc

```diff
@@ -0,0 +1,30 @@
+[id="create-postresql-secret_{context}"]
+
```

Comment:
This is not so significant and might not be worth correcting, but the "g" is missing in the file name (proc-create-postresql-secret.adoc) and [id=id="create-postresql-secret_{context}"]. Just an FYI.

Reply:
I think its ok so I'm going to leave it.